package.json

```diff
@@ -16,6 +16,7 @@
     "watch:dev": "nest start --watch",
     "watch:listen": "nest start --watch --config listener.json",
     "start:debug": "nest start --debug --watch",
+    "prestart:prod": "npm run build",
```

Comment:
Ahh, Oga what is this for?

Reply:
builds before starting, changes don't reflect unless you do that first